sbc_compassion/models/import_letter_line.py

```diff
@@ -27,6 +27,8 @@ class ImportLetterLine(models.Model):
                                      compute='_set_sponsorship_id')
     sponsorship_status = fields.Boolean(compute='_set_sponsorship_id',
                                         readonly=True)
+    person = fields.Boolean(compute='_set_sponsorship_id',
```

Comment:
What is this field for ? Isn't it the same as `sponsorship_status` ?
You could rename `_set_sponsorship_id` to `_find_sponsorship` since it is used for all fields related to sponsorship.